_includes/site-before-end.html

```diff
@@ -0,0 +1,3 @@
+{% for script in site.include_before_end_js %}
```

Comment:
As mentioned before, I think this would be more suited as just an empty html file. I'm guessing your use case would be to loop through a series of js files to add functionality to your site. If this was an empty file you could overwrite it with this very loop in your own project and place the variables mentioned in the config in your own config